core/shared/platform/common/posix/posix_thread.c

```diff
@@ -411,6 +415,9 @@ init_stack_guard_pages()
     uint32 guard_page_count = STACK_OVERFLOW_CHECK_GUARD_PAGE_COUNT;
     uint8 *stack_min_addr = os_thread_get_stack_boundary();
 
+    if (stack_min_addr == 0)
```

Comment:
Had better change to `stack_min_addr == NULL` or `!stack_min_addr`

Reply:
I changed it to this: `stack_min_addr == NULL`